Content.Server/SubFloor/TrayScannerSystem.cs

```diff
@@ -0,0 +1,24 @@
+using Content.Shared.Interaction;
+using Content.Shared.SubFloor;
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.SubFloor;
+
+public class TrayScannerSystem : SharedTrayScannerSystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<TrayScannerComponent, UseInHandEvent>(OnTrayScannerUsed);
+    }
+
+    private void OnTrayScannerUsed(EntityUid uid, TrayScannerComponent scanner, UseInHandEvent args)
```

Comment:
Given that the scanner works even when it is on the floor / not in your hands, maybe this should also have  an `ActivateInWorldEvent` so that you can just use 'E' while hovering over it.

Reply:
Also there is nothing in these interactions that prevents them from just being in the shared system, so you might as well have them in `SharedTrayScannerSystem`. It will make it easier whenever interactions are eventually fully predicted.